lib/schema_dev/github_actions.rb

```diff
@@ -0,0 +1,264 @@
+require 'pathname'
+require 'yaml'
+
+require_relative 'gemfile_selector'
+
+module SchemaDev
+  module GithubActions
+    extend self
+
+    WORKFLOW_FILE = ".github/workflows/prs.yml"
+
+    HEADER = <<~YAML
+      # This file was auto-generated by the schema_dev tool, based on the data in
+      #                 ./schema_dev.yml
+      # Please do not edit this file; any changes will be overwritten next time
+      # schema_dev gets run.
+    YAML
+
+    BASIC_WORKFLOW = {
+      name:        'CI PR Builds',
+      on:          {
+        push:         {
+          branches: %w[master],
+        },
+        pull_request: nil,
+      },
+      concurrency: {
+        group:                '${{ github.head_ref }}',
+        'cancel-in-progress': true,
+      }
+    }.freeze
+
+    BASIC_JOB = { 'runs-on': 'ubuntu-latest' }.freeze
+
+    BASIC_ENV = {
+      BUNDLE_GEMFILE: '${{ github.workspace }}/gemfiles/activerecord-${{ matrix.activerecord }}/Gemfile.${{ matrix.db }}'
+    }.freeze
+
+    FINISH_STEPS = [
+      {
+        name: 'Coveralls Finished',
+        if:   '${{ !env.ACT }}',
+        uses: 'coverallsapp/github-action@master',
+        with: {
+          'github-token':      '${{ secrets.GITHUB_TOKEN }}',
+          'parallel-finished': true,
+        }
+      }
+    ].freeze
+
+    STEPS = {
+      start:  [
+                {
+                  uses: 'actions/checkout@v2',
+                },
+                {
+                  name: 'Set up Ruby',
+                  uses: 'ruby/setup-ruby@v1',
+                  with: {
+                    'ruby-version':  '${{ matrix.ruby }}',
+                    'bundler-cache': true,
+                  },
+                },
+                {
+                  name: 'Run bundle update',
+                  run:  'bundle update',
```

Comment:
Why do we want to do a `bundle update` here?

Reply:
This is to ensure we are using the latest versions of the packages as everything is being "cached" from the above ruby install above. (though I guess I do need to test to see if it is actually caching the lock file)

Reply:
Why not simply disable the cache then? I don't think we depend on a lot of gems, so won't add too much overhead (especially since we're launching database instances via docker).